src/Transloadit.ts

```diff
@@ -184,6 +185,8 @@ export class Transloadit {
 
   private _lastUsedAssemblyUrl = ''
 
+  private _validateResponses = true
```

Comment:
```suggestion
  private _validateResponses = false
```

maybe let's also add a comment that, much like deprecation periods, we're not making these fatal just yet to allow for a period of schema adjustments without breaking everyone's install.

Reply:
we already have the option and the motivation behind it documented in README, or were you thinking about something else?